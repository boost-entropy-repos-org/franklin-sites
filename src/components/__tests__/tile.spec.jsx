import React from 'react';
import renderer from 'react-test-renderer';

import Tile from '../tile';

describe('Tile component', () => {
  test('should render', () => {
    const component = renderer.create(<Tile namespace="uniref" />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
