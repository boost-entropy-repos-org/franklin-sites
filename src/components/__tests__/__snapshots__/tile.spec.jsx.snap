// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile component should render 1`] = `
<div
  className="tile tile--uniref"
>
  <h3
    className="tile__header"
  >
    Tile title
  </h3>
  <p
    className="tile__content"
  >
    This is a short description of what the resource is/provides.
  </p>
</div>
`;

exports[`Tile component should render small tile 1`] = `
<div
  className="tile tile--uniref tile--small"
>
  <h3
    className="tile__header"
  >
    Tile title
  </h3>
</div>
`;
